### ISSUE TITLE
MNNConverter工具 torch.cat后进行conv操作就会出错,不需要torch.cat,就会成功

### ISSUE THREAD

**Issue by wuqingshan2010** [alibaba/MNN#496]
self.conv1 = torch.nn.Conv2d(2, 1, kernel_size, padding=padding, bias=False)
        self.tanh = torch.nn.Tanh()

    def forward(self, x):
        avg_out = torch.mean(x, dim=1, keepdim=True)  # 每一行的均值；
        max_out, _ = torch.max(x, dim=1, keepdim=True)  # 每一行的最大值；
        tt = torch.cat([avg_out, max_out], dim=1)
        t1 = self.conv1(tt)
        return self.tanh(t1) + 1

**Comment by wuqingshan2010** [comment]
inputParam->dformat = MNN::MNN_DATA_FORMAT_NCHW;跟改为inputParam->dformat = MNN::MNN_DATA_FORMAT_NC4HW4;
